submissions/Temu4/memory_pair_game/js/script.js

```diff
@@ -73,47 +67,37 @@ const rotateCard = event => {
 };
 
 const defaultCardsPosition = () => {
-  for (let i = 0; i < cards.length; i++) {
-    cards[i].classList.contains('rotate') ? cards[i].classList.remove('rotate') : null;
-  }
+  cards.forEach(card =>
+    card.classList.contains('rotate') ? card.classList.remove('rotate') : null
+  );
 };
 
 const hideAfterCorrectSelection = () => {
-  for (let i = 0; i < cards.length; i++) {
-    cards[i].classList.contains('rotate') ? cards[i].classList.add('hidden') : null;
-  }
+  cards.forEach(card => (card.classList.contains('rotate') ? card.classList.add('hidden') : null));
 };
 
 const showGameScore = () => {
```

Comment:
You can use arrow function simplier, without `return`, just inline.

Reply:
fix